app/code/Magento/Sales/Model/Order.php

```diff
@@ -1831,7 +1847,7 @@ public function getCreatedAtFormatted($format)
             new \DateTime($this->getCreatedAt()),
             $format,
             $format,
-            null,
+            $this->localeResolver->getDefaultLocale(),
```

Comment:
Nope, mock this dependency, the only thing to be asserted that we are passing concrete locale and not `null`.